### ISSUE TITLE
Code qusetion in offline.py

### ISSUE THREAD

**Issue by jwhao** [DongSky/few-shot-vit#7]
Hello, I'm very interested in your work and thank you for your open code.
I have a problem in your offline.py code.
On line 71:     bg_map = torch.full(pos_mask.size(), c, device=device)
actually, the c should be 64 in my understand from your paper "For background patches, the last positions of their pseudo label sij ∈ Rc+1 are one".
But, in the code， the c is 1, because on line 62: b, c, h, w = logits_max.size() .
So, is anything wrong?
